lib/pages/my_pixels/my_pixels.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/widgets.dart';
+import '../../widgets/scaffold.dart';
+import './widgets/header.dart';
+import './widgets/pixels.dart';
+
+class MyPixels extends StatelessWidget {
+  final bool myself;
+  final bool following;
+  final String photoURL;
+  final String name;
+  final String bio;
+  final int postsCount;
+  final int followersCount;
+  final int followingCount;
+  final List<String> pixels;
+
+  MyPixels(
+      {this.myself = true,
+      this.following = false,
+      @required this.photoURL,
+      @required this.name,
+      @required this.bio,
+      @required this.postsCount,
+      @required this.followingCount,
+      @required this.followersCount,
+      @required this.pixels});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+          child: Column(
+        children: [
+          Header(
+            following: following,
+            myself: myself,
+            photoURL: photoURL,
+            name: name,
+            bio: bio,
+            postsCount: postsCount,
+            followersCount: followersCount,
+            followingCount: followingCount,
+          ),
+          Pixels(pixels: pixels, total: postsCount),
+        ],
+      )),
```

Comment:
The wrong indentation (discussion above) is being caused by this line. For dartfmt to properly format the code, it needs some trailing commas after the properties (even though it's the last property).


So for correct formatting of the `Column` code, this line should be:
```dart
),),
````


Further explanation: dartfmt works that way since sometimes you don't want it to split your code in several lines. For instance, check line 44, if you put a comma after `total: postsCount` (before the closing parenthesis) it would put each constructor  parameter on its own line. So comma positioning has an impact on formatting to allow flexibility.


More info here: https://flutter.dev/docs/development/tools/formatting#using-trailing-commas

Reply:
Very important to know! Thanks to share it with us!